android/src/main/java/com/rudderstack/react/android/RNRudderSdkModule.java

```diff
@@ -88,6 +89,21 @@ public void screen(String event, ReadableMap properties, ReadableMap options) {
                 .build());
     }
 
+    @ReactMethod
+    public void putAndroidDeviceToken(String token) {
+        if (rudderClient == null) return;
+        RudderContext rudderContext = rudderClient.getRudderContext();
+        if (rudderContext != null && token != null) {
```

Comment:
Check for empty token as well. Better to put `TextUtils.isEmpty(token)`.

Reply:
done